client/src/app/components/home/home.component.html

```diff
@@ -143,6 +143,81 @@ <h2>Create your next Project with us</h2>
   </div>
 </div>
 
+<div class="sec-5">
+  <div class="container">
+    <div class="d-md-flex align-items-end justify-content-between w-100">
```

Comment:
add play cards medium flex and small screens block

Reply:
okay